invenio_app/ext.py

```diff
@@ -44,13 +48,19 @@ def init_app(self, app, **kwargs):
         """
         # Init the configuration
         self.init_config(app)
-        # Enable Rate limiter
-        self.limiter = Limiter(app, key_func=get_ipaddr)
+
         # Enable secure HTTP headers
         if app.config['APP_ENABLE_SECURE_HEADERS']:
             self.talisman = Talisman(
                 app, **app.config.get('APP_DEFAULT_SECURE_HEADERS', {})
             )
+
+        # Enable Rate limiter
+        self.limiter = Limiter(
```

Comment:
Add a comment here, explaining why this should be initialized after Talisman (for historical reasons)